frontend/src/components/CancelTransferBtn.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {useContractWrite} from "@starknet-react/core";
+import {TREASURY_ADDRESS} from "../lib/config";
+import {Contract} from "starknet";
+import TreasuryABI from "../lib/treasury_abi.json";
+
+interface CancelTransferButtonProps {
+    transferId: number;
+}
+
+const CancelTransferBtn: React.FC<CancelTransferButtonProps> = ({transferId}) => {
+
+    const contract = new Contract(TreasuryABI, TREASURY_ADDRESS)
+
+    const { writeAsync: write_yes , error } = useContractWrite({
```

Comment:
No reason to rename the `writeAsync` to `write_yes` here, especially since the new variable name doesn't make sense here.